### PR TITLE
Add new app icons

### LINKED ISSUE #5: better app icon?
I quickly crafted https://github.com/nextcloud/pdf_converter/blob/master/img/app-dark.svg which isn't terribly awesome. Office documents, including various things in plain text, and also images, can be converted to PDF, automagically. I tried to depict it with an arrow pointing towards PDF. At least the arrow's to thin… mayber there are better ideas, too.

may i ask one of @nextcloud/designers ?

### PR DESCRIPTION
**Pull request by juliushaertl**
Fixes #5

### CODE REVIEW

File: img/app-dark.svg [hunk 1]
```diff
@@ -1,82 +1 @@
...
...0.61838 0.61838 0 0 0 0.86992 7.91e-5l0.86886-0.8687a0.61838 0.61838 0 0 0 0.0011543-0.88702 0.61838 0.61838 0 0 0-0.67634-0.12303c0.04094-0.86013-0.27221-1.7117-0.89472-2.3343zm-3.3067 1.0814a0.61838 0.61838 0 0 0-0.015967-0.01364l-0.86984-0.87a0.61838 0.61838 0 0 0-0.042126-0.04213 0.61838 0.61838 0 0 0-0.82551 0.04205l-0.87 0.86984a0.61838 0.61838 0 0 0 0.66145 1.0237c-0.024276 0.84049 0.29182 1.6675 0.90045 2.2762 0.78035 0.78052 1.9205 1.0863 2.9866 0.80073a0.51533 0.51533 0 1 0-0.27202-0.99408c-0.71208 0.19075-1.4669-0.011716-1.988-0.53306-0.45484-0.45491-0.65183-1.0905-0.57258-1.7183l0.018216 0.018221a0.61843 0.61843 0 0 0 0.88935-0.85959z" stroke-width=".68342"/><path d="m31.219 0.33675v0.00113h-6.9286v1.3295l6.9286 0.036145c0.0026-1.821e-4 0.0053 2.074e-4 0.0079 0 0.0053-4.166e-4 0.01058-0.00137 0.01581-0.00113 0.65203-0.00106 1.1749 0.44619 1.1867 1.0392 0.0108 0.5673-0.60099 1.0888-1.3381 1.0019l-0.0013-0.79858-1.6753 1.5203 1.7016 1.4481-0.0013-0.8031c1.419 0.06127 2.9112-0.90236 2.9081-2.3709-0.0029-1.3197-1.2547-2.4007-2.7961-2.4014-0.0023-1e-6 -0.0043-0.00113-0.0066-0.00113z" stroke-width=".462"/><rect x="31.116" y="-1.6777" width="4.3279" height="7.5909"/></g></svg>
```
Comment (pixelipo) [inline img/app-dark.svg]: Please remove the grouping `<g>` element (fill is also not needed).

I would suggest removing stroke as well, but that's a bit more work.

**Comment by pixelipo** [comment]
@juliushaertl why two icons? `icon-white` should do the job?

**Comment by juliushaertl** [comment]
> Please remove the grouping <g> element (fill is also not needed).

I thought, those might be needed when using the icon cacher at some point, so it can be properly replaced with a different color?

> why two icons? icon-white should do the job?

We need the white icon as the general app icon and the dark one is used for the settings section (which doesn't use icon classes).

**Comment by pixelipo** [comment]
black icon should always be the default one and the "source". Therefore, the general app icon should use the icon-white class on top of the default version.

**Comment by juliushaertl** [comment]
The server still requires a white icon file for every app to build the menu. We need to change that before we can cleanup the icons in apps.

**Pull request merged**